### ISSUE TITLE
exec: "zip": executable file not found in $PATH

### ISSUE THREAD

**Issue by akujinnoninjin** [hakwerk/labca#18]
### Objective

Fresh install, attempting to download the certs as a bundled zip via _WebUI_ > _Manage_ > _Certificates_

### Result
I receive a zip file of approximately the right size, but it's unopenable. 
7zip "cannot open as archive", ``unzip`` on LabCA host gives
```
Archive:  ./labca_certificates.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of ./labca_certificates or
        ./labca_certificates.zip, and cannot find ./labca_certificates.ZIP, period.
```

### Logs
LabCA log only shows:
``2021/05/29 14:24:45 errorHandler: exec: "zip": executable file not found in $PATH``

### Attempted Fixes / Workarounds
- Confirmed that ``zip`` exists in ``/usr/bin``, and that ``/usr/bin`` is in ``$PATH`` _Edit: On the *host*_
- ``apt install zip`` hasn't made any difference
- Downloading as pfx and manually zipping works fine

### Environment
- Clean install of Debian 10 in an LXC Container. Can be reproduced in full VM.

**Comment by akujinnoninjin** [comment]
``zip`` is missing from the labca container, running ``docker exec boulder_labca_1 apk install zip -y`` resolves the issue.

**Comment by hakwerk** [comment]
This should be fixed now in the latest release (v21.06)
